client/app/scripts/components/node-details/node-details-table.js

```diff
@@ -188,10 +261,11 @@ export default class NodeDetailsTable extends React.Component {
                   colStyles={styles}
                   columns={columns}
                   onClick={onClickRow}
-                  onMouseLeaveRow={onMouseLeaveRow}
-                  onMouseEnterRow={onMouseEnterRow}
+                  onMouseEnter={() => { this.onMouseEnterRow(index, node); }}
```

Comment:
Best practice here would be to pass the index and node as props to the row, and then call the props.onMouseEnter from the row with props.node and props.index. Otherwise a new closure is generated on every row render.